lib/doorkeeper/oauth/invalid_request_response.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module OAuth
+    class InvalidRequestResponse < ErrorResponse
```

Comment:
Introduce `InvalidRequestResponse` to handle `invalid_request`
- I added explanation for `invalid_request` by two following variable: `missing_parameter` (for missing required parameter) and `invalid_request_reason` (for reason other than missing required parameter: happen at token_introspection endpoint and, authorization_code_request)